packages/reports/addon/components/report-actions/multiple-format-export.ts

```diff
@@ -122,7 +134,12 @@ export default class MultipleFormatExport extends ReportActionExport {
 
     try {
       if (exportType === 'CSV') {
-        yield taskFor(super.downloadTask).perform();
+        const serializedRequest = this.args.model.request.serialize() as RequestV2;
+        let url = yield taskFor(super.downloadTask).perform();
+        if (getDataSource(serializedRequest.dataSource).type === 'bard') {
```

Comment:
We don't want to add datasource specific logic in the components, we should update the `downloadTask` task in `export.ts` to pass a `filename: ...` attribute
https://github.com/yavin-dev/framework/blob/166b2a4a4139c9575003ec0074857164a330e6eb/packages/reports/addon/components/report-actions/export.ts#L62-L65
then update the options type of `getDownloadURL` to allow for `filename: string` so that each adapter can handle it
https://github.com/yavin-dev/framework/blob/e167311615defcf91944df3a9699a8b454c9f870/packages/data/addon/services/navi-facts.ts#L77-L81

Reply:
Modified the logic into export.ts, I have added fileName in optionsType.

Reply:
Did you forget to push your changes?